Options.py

```diff
@@ -727,7 +729,11 @@ def verify_keys(cls, data):
                 raise Exception(f"Found unexpected key {', '.join(extra)} in {cls}. "
                                 f"Allowed keys: {cls.valid_keys}.")
 
-    def verify(self, world, player_name: str, plando_options) -> None:
+    if typing.TYPE_CHECKING:
+        from Generate import PlandoSettings
+        from worlds.AutoWorld import World
+
+    def verify(self, world: World, player_name: str, plando_options: PlandoSettings) -> None:
```

Comment:
this seems wrong. `plando_options: PlandoSettings` should be an undefined reference to `PlandoSettings` when not type-checking, so you want to leave the `"` in

Reply:
so this?
```
if typing.TYPE_CHECKING:
    typing
def verify(self, world: "World", player_name: str, plando_options: "PlandoSettings")
```

Reply:
It's kind of strange to put `if` in a class declaration scope. I don't think I've seen that before. I would have guessed it might be a syntax error.

Reply:
i originally had it without the `if` and the class declarations in `"` but the linter complains.